.github/workflows/ci.yml

```
@@ -0,0 +1,71 @@
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+
+jobs:
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)